src/web/demo.rkt

```diff
@@ -30,10 +30,10 @@
   (λ (x)
     (and (not (*demo-output*))
          (let ([m (regexp-match #rx"^([0-9a-f]+)\\.[0-9a-f.]+" x)])
-           (and m (hash-has-key? *completed-jobs* (second m))))))
+           (and m (completed-job? (second m))))))
   (λ (x)
     (let ([m (regexp-match #rx"^([0-9a-f]+)\\.[0-9a-f.]+" x)])
-      (hash-ref *completed-jobs* (if m (second m) x)))))
+      (remove-me (if m (second m) x)))))
```

Comment:
@pavpanchekha Any suggestions how to remove this? It's returning the completed job object/value. Which I think breaks the abstraction boundary I'm trying to build.

From what I understand this function takes a random string and checks if it's a valid completed job then produces a url from it. I'm a bit lost in the functional nature of this call.

Reply:
NVM was just confused